### ISSUE TITLE
missing tags in drop down menu

### ISSUE THREAD

**Issue by nh357** [ucam-cl-dtg/scooter#103]
Issue found in 3aa3355b83e800980c602cfab5f1548b15f81d0f

The drop down tag menu  under the metadata tab in files on the editor seem to have disappeared. It just comes up with <new tag>.

**Comment by daviesian** [comment]
Seems to be working again now. Reopen if not.

**Comment by nh357** [comment]
Worked last week, but not working today on either Safari or Chrome from my home.

**Comment by daviesian** [comment]
It works here.

If you're at home, make sure you can see staging.isaacphysics.org. You may need to login with raven for that.

**Comment by nh357** [comment]
Yes, I can. It’s weird that the tags don’t work when not n the uni network.

Nicki

**Comment by daviesian** [comment]
Does this work for you?

https://staging.isaacphysics.org/api/content/tags

**Comment by nh357** [comment]
I get a list of the tags and they seem to all be there. 

Nicki
